src/udf/js_to_si.c

```
@@ -0,0 +1,276 @@
+
+					// wrap in SIValue
+					ret = SI_Node (_n) ;
+					SIValue_SetAllocationType (&ret, M_SELF) ;
+				}
+
+				else if (cid == js_edge_class_id) {
+					Edge *e = JS_GetOpaque (val, js_edge_class_id) ;
+
+					// clone edge
+					Edge *_e = rm_malloc (sizeof (Edge)) ;
+					memcpy (_e, e, sizeof (Edge)) ;
```

Cppcheck (2.18.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)